owncloudApp/src/main/java/com/owncloud/android/presentation/ui/files/filelist/MainFileListFragment.kt

```diff
@@ -220,12 +223,12 @@ class MainFileListFragment : Fragment(), SortDialogListener, SortOptionsView.Sor
             layoutManager.spanCount = ColumnQuantity(requireContext(), R.layout.grid_item).calculateNoOfColumns()
         }
 
-        setFooterCorrectly(viewType == ViewType.VIEW_TYPE_LIST)
+        //setFooterCorrectly(viewType == ViewType.VIEW_TYPE_LIST)
 
         fileListAdapter.notifyItemRangeChanged(0, fileListAdapter.itemCount)
     }
 
-    private fun setFooterCorrectly(isListMode: Boolean) {
+    /*private fun setFooterCorrectly(isListMode: Boolean) {
```

Comment:
Is this code not needed since now? Can we remove it and the places where this method is asked for?